builder/dockerfile/internals.go

```diff
@@ -242,13 +242,13 @@ func (b *Builder) download(srcURL string) (fi builder.FileInfo, err error) {
 	if err != nil {
 		return
 	}
+	defer tmpFile.Close()
```

Comment:
(Apply this comment to everywhere there is an explicit `.Close()` before a `Chtimes` call)